### PR TITLE
FEAT A WORKING PAGE THAT LET YOU CREATE YOUR OWN RECIPE 

### PR DESCRIPTION
**Pull request by QuentinVerwaerde**

### CODE REVIEW

**Review by Mewlyscin — APPROVED** [review]

File: backend/src/controllers/recipeControllers.js [hunk 1]
```diff
@@ -52,12 +52,24 @@ const edit = async (req, res, next) => {
 
 const add = async (req, res, next) => {
   try {
-    const newRecipe = await tables.recipe.create(req.body);
+    const { info, ingredients, steps } = req.body;
+    // const recipeInformations = req.body.info;
```
Comment (PierreA44) [inline backend/src/controllers/recipeControllers.js:56]: you must delete this 3 lines

File: frontend/src/components/CommentForm.jsx [hunk 2]
```diff
@@ -14,7 +12,7 @@ export default function CommentForm({ recipeID, isValidated, setIsValidated }) {
   } = useForm({
     defaultValues: {
       recipe_id: recipeID,
-      user_id: auth.id,
+      // user_id: auth.id,
```
Comment (PierreA44) [inline frontend/src/components/CommentForm.jsx:15]: You must add this line to post a comment

File: frontend/src/components/RecipeInformationForm.jsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+/* eslint-disable react/jsx-props-no-spreading */
+
+export default function RecipeInformationForm({ setSelectedInformations }) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  //   {
+  //   defaultValues: {
```
Comment (PierreA44) [inline frontend/src/components/RecipeInformationForm.jsx:13]: again some lines to delete

File: frontend/src/components/RecipeInformationForm.jsx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    "https://i.ibb.co/dWCZjNS/plathealthy.jpg"
+  );
+  const [formIsValidated, setFormIsValidated] = useState(false);
+  const onSubmit = (data) => {
+    setSelectedInformations(data);
+    setFormIsValidated(true);
+  };
+
+  // axios get sur type de recette
+  // et l'incorporer
```
Comment (PierreA44) [inline frontend/src/components/RecipeInformationForm.jsx:27]: ?

**Comment by PierreA44** [comment]
i'm waiting your last modification to validate the global PR !

**Review by PierreA44 — APPROVED** [review]
Good job

**Review by ioayoub — APPROVED** [review]

**Review by akornne — APPROVED** [review]

**Pull request merged**